api/v1/ytsaurus_types.go

```diff
@@ -734,8 +734,9 @@ type UpdateStatus struct {
 // YtsaurusStatus defines the observed state of Ytsaurus
 type YtsaurusStatus struct {
 	//+kubebuilder:default:=Created
-	State      ClusterState       `json:"state,omitempty"`
-	Conditions []metav1.Condition `json:"conditions,omitempty"`
+	State              ClusterState       `json:"state,omitempty"`
+	Conditions         []metav1.Condition `json:"conditions,omitempty"`
+	ObservedGeneration int64              `json:"observedGeneration,omitempty"`
```

Comment:
Probably it needs '//+optional' annotation here
I'm not shure that "omitempty" provides the same effects for code generation.

Reply:
Nevermind. New new required fields in scheme - should be fine as is.